third_party/terraform/resources/resource_storage_bucket.go

```diff
@@ -844,7 +844,7 @@ func resourceStorageBucketStateImporter(d *schema.ResourceData, meta interface{}
 
 func expandCors(configured []interface{}) []*storage.BucketCors {
 	corsRules := make([]*storage.BucketCors, 0, len(configured))
-	if len(corsRules) == 0 {
+	if len(configured) == 0 {
```

Comment:
This check can be moved above the line above. Currently we create the `corsRules` variable but then don't use it if `len(configured) == 0`

Reply:
done